src/components/NavigationReserve.js

```diff
@@ -0,0 +1,122 @@
+import React, { useState } from 'react';
+/* Install pure-react-carousel using -> npm i pure-react-carousel */
+
+const NavigationReserve = () => {
+  const current = new Date();
+  const date = `${current.getDate()}/${current.getMonth() + 1}/${current.getFullYear()}`;
+  const [show, setShow] = useState(false);
+  return (
+    <div>
+      <div className="flex items-center justify-between w-full p-6 border border-transparent rounded-r shadow bg-menu xl:hidden ">
```

Comment:
- [ ] The page in tablet view is cut out. it would be great if you add same style for mobile in the tablet view
![Screenshot_20220829_125048](https://user-images.githubusercontent.com/25227022/187195484-aaa9df19-28b6-438e-8851-526c4beb980a.png)